### ISSUE TITLE
Chef-12 reports resource cloning issue when also using chef-client cookbook

### ISSUE THREAD

**Issue by sspans** [chef-cookbooks/omnibus_updater#118]
Resource cloning error reported by chef 12.12.15 when also using the chef-client cookbook.

```
[2016-07-18T09:20:42+02:00] WARN: Cloning resource attributes for service[chef-client] from prior resource (CHEF-3694)
Previous service[chef-client]: /var/chef/cache/cookbooks/omnibus_updater/recipes/installer.rb:28:in `from_file'
Current  service[chef-client]: /var/chef/cache/cookbooks/chef-client/recipes/systemd_service.rb:30:in `from_file' at /var/chef/cache/cookbooks/chef-client/recipes/systemd_service.rb:30:in `from_file' at /opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-12.12.15/lib/chef/event_dispatch/dispatcher.rb:43:in `call'
```

**Comment by sspans** [comment]
I've submitted a fix in pull #120 - but this fails due to unrelated rubocop style warnings.

**Comment by sspans** [comment]
With another two commits the unrelated foodcritic and rubocop warnings are gone too.
